invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -103,8 +103,12 @@ export class RecordCitationField extends Component {
 
     // convert links in text to clickable links (ignoring punctuations at the end)
     const escapedCitation = _escape(citation); // escape html characters
-    const urlRegex = /(https?:\/\/[^\s,;]+(?=[^\s.,;]*\S))/g;
+    const urlRegex = /(https?:\/\/[^\s,;]+(?=[^\s.,;]*))/g;
```

Comment:
```suggestion
    const urlRegex = /(https?:\/\/[^\s,;]+(?=[^\s,;]*))/g;
```

I'm testing and I don't think this second fullstop is doing anything

Reply:
or actually I'm not sure what the positive lookahead is doing?

Reply:
You're right, I tested with the following strings and it doesn't make a difference:

```
Zenodo. https://doi.org/10.5281/zenodo.1234
Zenodo. https://doi.org/10.5281/zenodo.1234 some text
Zenodo. https://doi.org/10.5281/zenodo.1234.
Zenodo. https://doi.org/10.5281/zenodo.1234. some text
Zenodo. https://doi.org/10.5281/zenodo.1234,
Zenodo. https://doi.org/10.5281/zenodo.1234, some text
Zenodo. https://doi.org/10.5281/zenodo.1234;
Zenodo. https://doi.org/10.5281/zenodo.1234; some text
```